cmd/benchmark.go

```diff
@@ -309,13 +309,14 @@ func rallyCommandAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	signal.Enable()
+	ctx, stop := signal.Enable(cmd.Context(), logger.Info)
+	defer stop()
```

Comment:
@mrodm enabling by now signal handling in the same places as before, for easier control of changes.